cupy/cuda/stream.pyx

```diff
@@ -423,6 +423,38 @@ class _BaseStream:
         except RuntimeError:  # can be RuntimeError or CUDARuntimeError
             raise
 
+    def get_stream_flags(self):
```

Comment:
How about making it a property with name `flags` for simplicity? It could be accessed as `stream.flags` where `stream` is an `_BaseStream` instance.

Reply:
Made the changes!